testdata/fixtures/group/descendant_repositories_request.json

```diff
@@ -0,0 +1,8 @@
+{
+    "query": "query($after:String!$first:Int!$group:ID!){account{group(id: $group){descendantRepositories(after: $after, first: $first){nodes{archivedAt,createdOn,defaultAlias,defaultBranch,description,forked,htmlUrl,id,languages{name,usage},lastOwnerChangedAt,name,organization,owner{alias,id},private,repoKey,services{edges{atRoot,node{id},paths{href,path},serviceRepositories{baseDirectory,displayName,id,repository{id,defaultAlias},service{id}}},pageInfo{hasNextPage,hasPreviousPage,startCursor,endCursor},totalCount},tags{nodes{id,key,value},pageInfo{hasNextPage,hasPreviousPage,startCursor,endCursor},totalCount},tier{alias,description,id,index,name},type,url,visible},pageInfo{hasNextPage,hasPreviousPage,startCursor,endCursor}}}}}",
+    "variables": {
+      "first": 100,
+      "group": "Z2lkOi8vb3BzbGV2ZWwvTmFtZXNwYWNlczo6R3JvdXAvMTI",
+      "after": ""
```

Comment:
is this the right value?

Reply:
Nope, looks like I forgot to go back and update this.